target/linux/realtek/files-5.10/drivers/clk/realtek/Makefile

```diff
@@ -0,0 +1,2 @@
+# SPDX-License-Identifier: GPL-2.0-only
+obj-$(CONFIG_COMMON_CLK_RTL83XX) += clk-rtl83xx.o clk-rtl838x-sram.o clk-rtl839x-sram.o
```

Comment:
is it worth-wile considering loading this blob via /lib/firmware instead? Now you are adding unused code (blobs) in your binary for no reason, wasting some space in the binary.

Reply:
We will waste space and add a TODO